### PR TITLE
Add java texture examples

### PR DESCRIPTION
**Pull request by chakravarthi589**
@ajpmaclean : Please review.

### CODE REVIEW

File: src/Java/Texture/TexturedSphere.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+
+  public static void main(String args[]) 
+  {
+    //parse command line arguments
+    if (args.length < 2) 
+    {
```
Comment (ajpmaclean) [inline src/Java/Texture/TexturedSphere.java]: This should be: `if (args.length < 1)`

File: src/Java/Texture/TexturedSphere.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      System.err.println("Usage: java -classpath ... Filename(.png) translate e.g earthTexture.png 0 ");
+      return;
+    }
+    double translate[] = new double[3];
+    String inputFilename = args[0];
+    
+    if (args.length > 2)
+    {
```
Comment (ajpmaclean) [inline src/Java/Texture/TexturedSphere.java]: This should be: if (args.length > 1)

File: src/Java/Texture/TexturedSphere.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  }
+  // -----------------------------------------------------------------
+
+  public static void main(String args[]) 
+  {
+    //parse command line arguments
+    if (args.length < 2) 
+    {
+      System.err.println("Usage: java -classpath ... Filename(.png) translate e.g earthTexture.png 0 ");
+      return;
```
Comment (ajpmaclean) [inline src/Java/Texture/TexturedSphere.java]: Change this line to:
``` java
System.err.println("Usage: java -classpath ... Filename(.png or .ppm) translate e.g earth.ppm 0 ");
```

**Comment by ajpmaclean** [comment]
`earth.ppm` is actually used in the Cxx example for testing.

File: src/Java/Texture/TexturedSphere.java (hunk 1 above)
Comment (chakravarthi589) [reply src/Java/Texture/TexturedSphere.java]: Have corrected.

File: src/Java/Texture/TexturedSphere.java (hunk 2 above)
Comment (chakravarthi589) [reply src/Java/Texture/TexturedSphere.java]: Have corrected.

File: src/Java/Texture/TexturedSphere.java (hunk 3 above)
Comment (chakravarthi589) [reply src/Java/Texture/TexturedSphere.java]: Have corrected.

**Comment by chakravarthi589** [comment]
Please review.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**